includes/class-sensei-modules.php

```diff
@@ -1258,7 +1258,7 @@ public function enqueue_styles() {
 		$disable_styles = apply_filters( 'sensei_disable_styles', $disable_styles );
 
 		if ( ! $disable_styles ) {
-			wp_register_style($this->taxonomy . '-frontend', esc_url($this->assets_url) . 'css/modules-frontend.css', Sensei()->version );
+			wp_register_style($this->taxonomy . '-frontend', esc_url($this->assets_url) . 'css/modules-frontend.css', false, Sensei()->version );
```

Comment:
May be slightly better to pass an empty array instead of `false` for the dependencies parameter since [passing a boolean gets converted to an empty array](https://core.trac.wordpress.org/browser/tags/4.9/src/wp-includes/class-wp-dependency.php#L77) anyway.

Reply:
Yes, why not :)